src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -43,234 +57,55 @@ public static function register($mode = 0)
             return;
         }
 
-        $UtilPrefix = class_exists('PHPUnit_Util_ErrorHandler') ? 'PHPUnit_Util_' : 'PHPUnit\Util\\';
+        self::computeMode($mode);
```

Comment:
Computing the mode should be kept lazy: it allows registering the bridge and configure the mode a bit later before any deprecation is thrown. That's an important behavior we should keep.

Reply:
Ok, I introduced a new `$rawMode` static to keep the value until it is necessary to compute the actual mode.